pkgs/top-level/python-packages.nix

```diff
@@ -5408,6 +5408,8 @@ in {
 
   pysolr = callPackage ../development/python-modules/pysolr { };
 
+  pysonic = callPackage ../development/python-modules/pysonic { };
```

Comment:
Not sure why you named the path `pysonic` instead of `py-sonic`.

Reply:
You're right. Changed.